### PR TITLE
replace authCapture with `capture`, and capture as `captureAuth`

### PR DESCRIPTION
**Pull request by TomK**

### CODE REVIEW

File: psp/captureauth_test.go [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"testing"
+)
+
+func TestCapture(t *testing.T) {
```
Comment (fobilow) [inline psp/captureauth_test.go:11]: This should probably be called TestCaptureAuth

File: psp/captureauth_test.go [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+	h := testCaptureHandler{c: testHandlerCredentials{"abc", "123"}}
+
+	con := NewTestConnection(h)
+	req := CaptureAuthRequest{Amount: NewAmount(123, "USD")}
```
Comment (fobilow) [inline psp/captureauth_test.go:15]: can you set the AuthorizeID in the test just to demonstrate that it is required

File: psp/captureauth_test.go (hunk 1 above)
Comment (TomK) [reply psp/captureauth_test.go]: done

File: psp/captureauth_test.go (hunk 2 above)
Comment (TomK) [reply psp/captureauth_test.go]: done

**Review by fobilow — APPROVED** [review]

**Pull request merged**